### DISCUSSION TITLE
Graph Dashboard is Loading endlessly in V2.0

### DISCUSSION THREAD

**Discussion by Eurofins2023** [plausible/analytics#2922] [Self-Hosted Support]
I have updated the plausible 2.0 version but the graph is loading endlessly.

Can anyone please debug here ?

**Comment by ruslandoga** [comment]
👋 @Eurofins2023 

Could you please check if the issue described in https://github.com/plausible/analytics/discussions/2896#discussioncomment-5850822 is in any way similar?
